src/accel/LocalTransporter.cc

```diff
@@ -124,6 +128,38 @@ void LocalTransporter::SetEventId(int id)
     track_counter_ = 0;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Set the event ID and reseed the Celeritas RNG at the start of an event.
+ */
+void LocalTransporter::InitializeEvent(int id)
+{
+    CELER_EXPECT(*this);
+    CELER_EXPECT(id >= 0);
+
+    this->SetEventId(id);
```

Comment:
Since ensuring reproducibility is a "bug fix", I think we can change the behavior so we *always* initialize at the start of an event. I like your name change that reflects the initialization cost, so what about having `SetEventId` be a `[[deprecated]]` inline alias to `InitializeEvent` (update the comment to say `to be removed in v1.0`), and replacing this call with the `SetEventId` code?

Reply:
Good call.